src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -52,6 +53,27 @@
 }
 
 
+def configure_dynatrace_exporter(
```

Comment:
I split it up in [82e7561](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/82e756162b8a90cbe1e254e2ce99afa57ad7cc2a). I previously held off on it because the diff gets huge with things like that.